### PR TITLE
Add the pictures and fix the broken links

### PR DESCRIPTION
**Pull request by jpark0123**
Signed-off-by: Jihoon Park <jpark0@samsung.com>

### Change Description ###

- Add the pictures for "Installing Drivers" for 5.0
- Correct indent and typos
- Fix the broken links, e.g. flashing_images..

### Bugs Fixed ###

N/A

### API Changes ###

N/A

### CODE REVIEW

File: docs/iot/getting-started/artik-no-sd-4.0.md [hunk 1]
```diff
@@ -263,21 +263,22 @@ For detailed information, see <https://wiki.tizen.org/Booting_Tizen_From_eMMC_On
 
 1.  Connect Smart Development Bridge (SDB) as described in the previous section.
 2.  Install the connectivity drivers:
-        1.  Download the plugin zip file from the **ARTIK530(4.0) Plugin** section at <http://developer.samsung.com/tizendevice/firmware> and follow the instructions.
 
-        ![ARTIK530(4.0) Plugin section](media/wifi_driver_artik530.png)
+    1.  Download the plugin zip file from the **ARTIK530(4.0) Plugin** section at <http://developer.samsung.com/tizendevice/firmware> and follow the instructions.
 
-        2.  For the case of the Linux shell (Linux), run the `sh` script given in the instructions. For example:
+    ![ARTIK530(4.0) Plugin section](media/wifi_driver_artik530.png)
```
Comment (mijin85cho) [inline docs/iot/getting-started/artik-no-sd-4.0.md]: Would you please add more spaces here for the indentation as on artik-no-sd-5.0.md?

**Comment by huytaduc** [comment]
checkout the webpage https://docs.stg.tizen.org/staging/448

File: docs/iot/getting-started/artik-no-sd-4.0.md [hunk 2]
```diff
@@ -263,21 +263,22 @@ For detailed information, see <https://wiki.tizen.org/Booting_Tizen_From_eMMC_On
 
 1.  Connect Smart Development Bridge (SDB) as described in the previous section.
 2.  Install the connectivity drivers:
-        1.  Download the plugin zip file from the **ARTIK530(4.0) Plugin** section at <http://developer.samsung.com/tizendevice/firmware> and follow the instructions.
 
-        ![ARTIK530(4.0) Plugin section](media/wifi_driver_artik530.png)
+    1.  Download the plugin zip file from the **ARTIK530(4.0) Plugin** section at <http://developer.samsung.com/tizendevice/firmware> and follow the instructions.
 
-        2.  For the case of the Linux shell (Linux), run the `sh` script given in the instructions. For example:
```
Comment (jpark0123) [inline docs/iot/getting-started/artik-no-sd-4.0.md]: getting-started/artik-no-sd-4.0.md has been removed from b900d643899ed5ff660ce19612f3fe86a63f8c57 commit.

File: docs/iot/getting-started/artik-no-sd-4.0.md (hunk 1 above)
Comment (jpark0123) [reply docs/iot/getting-started/artik-no-sd-4.0.md]: getting-started/artik-no-sd-4.0.md has been removed from b900d64 commit.

File: docs/iot/getting-started/rpi3-4.0.md [hunk 3]
```diff
@@ -318,7 +317,7 @@ sudo apt-get install package-name
 
 ### Installing IoT Setup Manager
 
-Ensure that you have installed [Tizen Studio](tizen-studio-install.md).
+Ensure that you have installed [Tizen Studio]((#Installing-Tizen-Studio).
```
[the remark this answers is not in the document]
Comment (jpark0123) [reply docs/iot/getting-started/rpi3-4.0.md]: getting-started/rpi3-4.0.md has been removed from b900d64 commit.

File: docs/iot/getting-started/artik-sd-4.0.md [hunk 4]
```diff
@@ -288,7 +288,7 @@ sudo apt-get install package-name
 
 ### Installing IoT Setup Manager
 
-Ensure that you have installed [Tizen Studio](tizen-studio-install.md).
+Ensure that you have installed [Tizen Studio]((#Installing-Tizen-Studio).
```
[the remark this answers is not in the document]
Comment (jpark0123) [reply docs/iot/getting-started/artik-sd-4.0.md]: Done. Thanks.

[the remark this answers is not in the document]
Comment (jpark0123) [reply docs/iot/getting-started/artik-sd-4.0.md]: getting-started/artik-no-sd-4.0.md has been removed from b900d64 commit.

File: docs/iot/getting-started/rpi3-5.0.md [hunk 5]
```diff
@@ -321,7 +320,7 @@ sudo apt-get install package-name
 
 ### Installing IoT Setup Manager
 
-Ensure that you have installed [Tizen Studio](tizen-studio-install.md).
+Ensure that you have installed [Tizen Studio]((#Installing-Tizen-Studio).
```
[the remark this answers is not in the document]
Comment (jpark0123) [reply docs/iot/getting-started/rpi3-5.0.md]: Done. Thanks.

**Comment by jpark0123** [comment]
@mijin85cho, I addressed all your comment and upload patch by "git commit --amend" and "git push -f".

**Comment by huytaduc** [comment]
checkout the webpage https://docs.stg.tizen.org/staging/448

**Comment by jpark0123** [comment]
This PR has been merged to https://github.com/Samsung/tizen-docs/pull/456

**Pull request closed without merging**